charts/reform-scan-blob-router/values.yaml

```diff
@@ -27,6 +27,7 @@ java:
     DELETE_DISPATCHED_FILES_CRON: "0 0/10 * * * *"
     HANDLE_REJECTED_FILES_CRON: "0/30 * * * * *"
     REJECT_DUPLICATES_CRON: "0/30 * * * * *"
+    CHECK_NEW_ENVELOPES_CRON: "0 55 0 * * MON-FRI" # every hour at 55 minutes on weekdays
```

Comment:
it looks like once per day. also missing `?` on a day declaration rather than `*`

Reply:
Yes, looks like at 0:55 - once per day